templates/students/attendance.html

```diff
@@ -83,6 +83,21 @@ <h3 class="text-center">Students Attendance</h3>
 
 {% block scripts-sidebar %}
 
+<script>
+    function attendance (student_id) {
+        $.ajax({
+            url: "{% url 'students:ajax_student_attendance' %}",
+            data: {
+                'std_id': student_id,
+            },
+            dataType: 'json',
+            success: function (data) {
+
```

Comment:
Add a success toast here showing that the attendance was marked successfully. You may refer b4715eb8 to see how I implemented toast in your previous PR.

Also, add the `error` parameter to in this AJAX and toggle the checkbox to it's original state if an error occurs along with showing a toast displaying the failure message.